tox.ini

```diff
@@ -57,4 +57,5 @@ commands =
 # Excluding our upstream submodule, and our vendored f5 configuration script.
 
 ignore=F403,H303,R
-exclude=openstack-ansible/*,f5-config.py
```

Comment:
Yes, I don't think flake8 failures for f5-config.py are relevant to the purpose of this commit. I would suggest adding an issue to the backlog and it can be prioritised appropriately.